src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -653,6 +662,21 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
     this.updateResolutionStepValidationDetail()
   }
 
+  private async reRenderResolutionText (): Promise<void> {
```

Comment:
Not async. Does the method within need to be awaited?

Reply:
good catch, it used to be awaited.

Reply:
Also `onResolutionVisibilityChanged()` below.

Reply:
removed this function and moved logic into onResolutionVisibilityChanged